modules/mesaides/files/ma-web.conf

```diff
@@ -11,4 +11,11 @@ setuid main
 
 chdir /home/main/mes-aides-ui
 
-exec node server
+script
+    set -a
+    if [ -f /home/main/.secrets ]
+    then
+         . /home/main/.secrets
```

Comment:
Yes they are synonymous but only in `bash`not in shell. `source` is not working is an `upstart` job file.